.github/workflows/workflow.yaml

```diff
@@ -41,6 +41,7 @@ jobs:
           flake8 openghg/ --count --statistics
       - name: Install openghg
         run: |
+          pip install -r requirements.txt
```

Comment:
I think (as you pointed out) as these dependencies are essentially included within the setup.py we shouldn't need to add this line. The `pip install .` below should essentially cover including details from `requirements.txt`.

Reply:
Okay I took it out.